src/Requests/CollectionOperations/ListCollections.php

```diff
@@ -2,29 +2,34 @@
 
 namespace HelgeSverre\Milvus\Requests\CollectionOperations;
 
+use Saloon\Contracts\Body\HasBody;
 use Saloon\Enums\Method;
 use Saloon\Http\Request;
+use Saloon\Traits\Body\HasJsonBody;
 
 /**
  * List Collections
  *
  * Lists collections in a cluster.
  */
-class ListCollections extends Request
+class ListCollections extends Request implements HasBody
 {
-    protected Method $method = Method::GET;
+    use HasJsonBody;
+
+    protected Method $method = Method::POST;
 
     public function resolveEndpoint(): string
     {
-        return '/v1/vector/collections';
+        return '/v2/vectordb/collections/list';
     }
 
     public function __construct(
         protected ?string $dbName = null,
     ) {
+        $this->body()->setJsonFlags(JSON_FORCE_OBJECT);
```

Comment:
I had to set this flag, because when there isn't any parameter given, the defaultBody array is empty. And when Saloon sends it to Milvus, the default json encoded request body is `[]`. Milvus returns errors because it requires an object.